src/Products/CMFCore/tests/test_CookieCrumbler.py

```diff
@@ -254,6 +255,14 @@ def test_before_traverse_hooks(self):
         self.assertEqual(len(bt_removed.items()), 0)
 
 
+# compatibility
```

Comment:
Maybe adding the Zope versions which added `normalizeCookieParameterName`, so this hook could be removed in the future when older versions are no longe supported.